coldcard-cli/src/main.rs

```diff
@@ -782,6 +822,69 @@ fn handle(cli: Cli) -> Result<(), Error> {
             }
             println!("{}", xpub);
         }
+        Command::MiniscriptList => {
+            let resp = cc.miniscript_list()?;
+            for m in resp {
+                println!("{m}");
+            }
+        }
+        Command::MiniscriptDelete { name } => {
+            let descriptor_name = match DescriptorName::new(name.clone()) {
+                Ok(n) => n,
+                Err(e) => {
+                    println!("ERROR: Invalid miniscript descriptor name: {e:?}.");
+                    return Ok(());
+                }
+            };
+            if let Err(e) = cc.delete_miniscript(descriptor_name) {
+                println!("ERROR: Fail to delete descriptor {name}: {e:?}")
+            }
+        }
+        Command::MiniscriptPolicy { name } => {
+            let descriptor_name = match DescriptorName::new(name.clone()) {
+                Ok(n) => n,
+                Err(e) => {
+                    println!("ERROR: Invalid miniscript descriptor name: {e:?}.");
+                    return Ok(());
+                }
+            };
+            match cc.bip388_policy_get(descriptor_name) {
+                Ok(Some(p)) => println!("{p}"),
+                Err(e) => {
+                    println!("ERROR: Fail to get miniscript policy for name {name}: {e:?}")
+                }
+                _ => {
+                    println!("ERROR: No descriptor found with name {name}");
+                }
+            }
+        }
+        Command::Restore {
+            password,
+            path,
+            plaintext,
+            tmp,
+        } => {
+            if !path.exists() {
+                println!("ERROR: Path {} does not exists", path.to_str().unwrap());
+                return Ok(());
+            }
+            if !path.is_file() {
+                println!("ERROR: Path {} is not a file", path.to_str().unwrap());
+                return Ok(());
+            }
+            let mut file = match File::open(path.clone()) {
+                Ok(f) => f,
+                Err(e) => {
+                    eprintln!("ERROR: Fail to open file {}: {e}", path.to_str().unwrap());
+                    return Ok(());
+                }
+            };
+            let mut data = vec![];
+            let _ = file.read_to_end(&mut data);
+
+            println!("{password} {plaintext} {tmp}");
```

Comment:
Is this println necessary? Seems like it's printing some touchy information for no reason.